lib/network/tsnetwork_bwlimit.c

```diff
@@ -120,6 +134,40 @@ pokeone(struct bwlimit * l, double t, int op)
 	return (-1);
 }
 
+#ifdef TSNETWORK_PRINT_SPEED_SECONDS
+static void
+print_speed(struct timeval tnow)
+{
+	static struct timeval torig;
```

Comment:
I found it easier to reason about these two static variables when they were inside the `#ifdef`'d function.  You use static function variables infrequently (for example, in `getserveraddr()` in `netproto_connect.c`), but in this case the alternative was to add another top-level `#ifdef` section.